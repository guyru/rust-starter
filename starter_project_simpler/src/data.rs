// use anyhow::anyhow;
// use anyhow::Result as AnyResult;
use serde_derive::{Deserialize, Serialize};
use std::collections::HashMap;

#[derive(Debug, Serialize, Deserialize)]
pub struct Definitions {
    pub providers: HashMap<String, String>,
}

pub const CMD: &str = r#"hello"#;

#[cfg(test)]
mod tests {
    use super::*;
    use insta::assert_yaml_snapshot;
    use pretty_assertions::assert_eq;

    #[test]
    fn test_foo() {
        assert_eq!(CMD.len(), 5);
        assert_yaml_snapshot!(("one", "two"));
    }
}
