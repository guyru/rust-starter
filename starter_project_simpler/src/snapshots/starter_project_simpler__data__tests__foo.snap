---
source: starter_project_simpler/src/data.rs
expression: "(\"one\", \"two\")"
---
- one
- two

